test/Fondue.test.js

```diff
@@ -130,3 +130,12 @@ describe("hasAxes", () => {
 		expect(fondue.variable).toBeUndefined();
 	});
 });
+
+describe("chars", () => {
+	test("supports char", async () => {
```

Comment:
The strings passed to `describe` and `test` are used together to form output of the test runner. The output is not very meaningful now:

![Screenshot 2020-08-26 at 16 45 01](https://user-images.githubusercontent.com/304603/91318443-84bb7600-e7bb-11ea-8721-5c390a3dc6c0.png)

What do you think of naming this `supportedCharacters` and `returns characters of best of cmap`? Maybe also a quick test to verify that 65535 is indeed skipped.

Reply:
Great suggestion, thanks!